src/app/selectors.ts

```diff
@@ -235,3 +240,15 @@ export const registrationStepsSelector = createSelector(
 
 export const centralPhoneVerificationEnabledSelector = (state: RootState) =>
   state.app.centralPhoneVerificationEnabled
+
+export const numberVerifiedCentrallySelector = (state: RootState) => state.app.phoneNumberVerified
+
+export const phoneNumberVerifiedSelector = createSelector(
+  [
+    centralPhoneVerificationEnabledSelector,
+    numberVerifiedCentrallySelector,
+    numberVerifiedSelector,
+  ],
+  (centralPhoneVerificationEnabled, numberVerifiedCentrally, numberVerifiedDecentrally) =>
+    centralPhoneVerificationEnabled ? numberVerifiedCentrally : numberVerifiedDecentrally
```

Comment:
i'm wondering it could cause confusion for the user unless all our rewards program backends continue to check against central and decentral verification...

i'll think about it some more, i guess it's not clear how long the migration would take so maybe when the migration is running we need to know about the decentral verification status

Reply:
Ok I see your point, yes I was mainly thinking about the time when the migration hasn't happened yet. It should be quick though, just an API call, but like you mentioned it can fail (network issue mainly). So preferred to avoid confusing the user by showing state in the UI saying they aren't verified.
But maybe we should chat directly.